vim/vscode-jumplist.vim

```diff
@@ -23,6 +29,7 @@ augroup VscodeJumplist
 augroup END
 
 
+onoremap <C-o> <Cmd>call <SID>jumpToMark()<CR>
```

Comment:
Why `<C-o>` here? What about jumping with ``` `A ``` and `'A` ?

Reply:
Omg, sorry, just a stupid typo after merge. Fixed.